Makefile

```diff
@@ -24,28 +21,33 @@ endif
 CFLAGS += -I.
 CFLAGS += -std=c99 -Wall -pedantic
 
-
-all: $(TARGET)
+all: $(OBJ)
 
 asm: $(ASM)
 
 size: $(OBJ)
 	$(SIZE) -t $^
 
 .SUFFIXES:
-test: test_format test_dirs test_files test_seek test_truncate test_parallel \
-	test_alloc test_paths test_orphan test_move test_corrupt
-test_%: tests/test_%.sh
+test: $(TESTS)
+$(TESTS): $(OBJ)
+test_%: tests/test_%.sh FORCE
 ifdef QUIET
-	./$< | sed -n '/^[-=]/p'
+	@./$< | sed -n '/^[-=]/p'
```

Comment:
I'm going to have to at least steal some of these if that's ok with you!

Reply:
I have a separate pull-request pending after the #21 which includes this.

Reply:
How comfortable are you with git branches and rebasing? You could move test changes into a separate branch from doc-tweaks so it could be reveiwed and merged in parallel.

``` bash
# on master
$ git checkout -b parallel-tests
$ git rebase --onto f694b14 25fc42f parallel-tests
# now on branch parallel-tests separate from docs
$ git push origin parallel-tests
```